packages/wonder-blocks-tooltip/components/tooltip-bubble.test.js

```diff
@@ -0,0 +1,74 @@
+// @flow
+import * as React from "react";
+import * as ReactDOM from "react-dom";
+import {mount} from "enzyme";
+import TooltipBubble from "./tooltip-bubble.js";
+
+/**
+ * A little wrapper for the TooltipBubble so that we can provide an anchor
+ * element reference and test that the children get rendered.
+ */
+class BubbleTest extends React.Component<*, {ref: ?HTMLElement}> {
+    state = {
+        ref: null,
+    };
+
+    updateRef(ref) {
+        const actualRef = ref && ReactDOM.findDOMNode(ref);
+        if (actualRef && this.state.ref !== actualRef) {
+            this.setState({ref: ((actualRef: any): ?HTMLElement)});
+        }
+    }
+
+    render() {
+        const resultRef = (((
+            <div ref={(ref) => this.props.resultRef(ref)}>
+                This is a pretend string with a ref so we can detect it being
+                rendered
+            </div>
+        ): any): string);
+        return (
+            <div>
+                <div ref={(ref) => this.updateRef(ref)}>Anchor</div>
+                <TooltipBubble
```

Comment:
The tests should be written using `View` since people will be using `View` in production.

Reply:
Sadly, that creates a lot of heavy lifting with the refs since the popper.js stuff wants `HTMLElement` types.

In reality, this component will be internal to our other tooltip components, so I don't know that we gain anything by putting in the effort to use View here. Certainly, we should use public components for our public components (which for Tooltip, will be Tooltip and TooltipContent).